lib/oban/config.ex

```diff
@@ -13,7 +13,7 @@ defmodule Oban.Config do
   @type t :: %__MODULE__{
           dispatch_cooldown: pos_integer(),
           engine: module(),
-          get_dynamic_repo: nil | (-> pid() | atom()),
+          get_dynamic_repo: nil | (-> pid() | atom()) | {module(), atom(), list()},
```

Comment:
```suggestion
          get_dynamic_repo: nil | (-> pid() | atom()) | mfa(),
```

Reply:
unfortunately, `mfa()` is aliased to `{module(), atom(), arity()}` which isn't what we want here

Reply:
Indeed it is 😞